base/abstractarray.jl

```diff
@@ -343,6 +343,27 @@ function first(itr)
     x[1]
 end
 
+"""
+    first(v::AbstractVector, n::Integer)
+
+Get the first n elements of the vector, without going out-of-bounds.
```

Comment:
Maybe something like this?
```suggestion
Get the first `n` elements of vector `v`, or fewer elements if `v` is not long enough.
```